### DISCUSSION TITLE
Authorization and user management

### DISCUSSION THREAD

**Discussion by sandbox-apps** [prisma/prisma#8815] [Q&A] [answered]
Hello, I've seen this on the Prisma database features matrix reference. Is there any propose/discussion over this? I just want to learn of what it is. Try to grasp the idea on how it will be used on prisma

**Comment by ryands17** [comment, marked as answer]
@sandbox-apps 👋 

This is mostly regarding [Row Level Security](https://www.postgresql.org/docs/current/ddl-rowsecurity.html) offered by databases like Postgres. This can be easily done from the database end but Prisma Client doesn't support it yet.
https://github.com/prisma/prisma/issues/5128

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
